### PR TITLE
Fix tests of rendering net-att-def config

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

**Comment by pliurh** [comment]
This shall be merged before https://github.com/openshift/release/pull/13934

File: Makefile [hunk 1]
```diff
@@ -152,6 +152,9 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
+skopeo:
```
Comment (fedepaol) [inline Makefile:155]: What's the need for skopeo here?

Comment (pliurh) [reply Makefile:155]: skopeo is required by hack/env.sh

Comment (zshi-redhat) [reply Makefile:155]: Just in case, I think we don't assume local installation will work in a disconnected environment, right?

**Pull request merged**